bin/describe-feature.py

```diff
@@ -93,6 +93,8 @@ def reportGenomeFeature(features, name, alignment, maxSequenceLength, oneBased):
     print(f"    start: {absoluteStart + bool(oneBased)}")
     print(f"    stop: {absoluteStop}")
     print(f"    length (nt): {len(genomeNt.sequence)}")
+    print(f"    aligned (to ref) start: {alignedStart + bool(oneBased)}")
+    print(f"    aligned (to ref) stop: {alignedStop}")
```

Comment:
I got rid of the `bool(...)` since it is already a `bool`, thanks. It doesn't affect the stop vars because the stop site is the same, whether you have an inclusive 1-based view of the world or a closed-open 0-based Python view of the world.